docs/api/checkouts.rst

```diff
@@ -1,235 +1,560 @@
+            "_links":{
+                "self":{
+                    "href":"\/api\/v1\/channels\/1"
+                }
+            }
+        },
+        "shipping_address":{
+            "first_name":"Frederick D.",
+            "last_name":"Gregory",
+            "country_code":"US",
+            "street":"300 E St SW",
+            "city":"\u2019Washington",
+            "postcode":"DC 20546"
+        },
+        "billing_address":{
+            "first_name":"Frederick D.",
+            "last_name":"Gregory",
+            "country_code":"US",
+            "street":"300 E St SW",
+            "city":"\u2019Washington",
+            "postcode":"DC 20546"
+        },
+        "payments":[
+            {
+                "id":21,
+                "method":{
+                    "id":1,
+                    "code":"cash_on_delivery"
                 },
-                "created_at": "2014-12-15T14:11:32+0000",
-                "state": "checkout"
+                "amount":108787,
+                "state":"cart"
+            }
+        ],
+        "shipments":[
+            {
+                "id":21,
+                "state":"cart",
+                "method":{
+                    "code":"ups",
+                    "enabled":true
+                }
             }
-        }
-    ]
+        ],
+        "currency_code":"USD",
+        "locale_code":"en_US",
+        "checkout_state":"addressed"
+    }
 
-Response contains the proposed shipments and for each, it also has a list of shipping methods available.
+Shipping step
+-------------
 
-Next step is updating the order with the types of shipping method that we have selected.
-To do so, you need to call another PUT request, but this time with different set of parameters.
+When order contains the address information, we are able to determine the stock locations and available shipping methods.
+First we need to get information about available shipping methods to know our choices:
 
-You need to pass an id of shipping method for every id, you should obtain them in the previous request.
+Definition
+^^^^^^^^^^
 
 .. code-block:: text
 
-    PUT /api/v1/checkouts/44
+    GET /api/v1/checkouts/select-shipping/{id}
+
++---------------+----------------+--------------------------------------+
+| Parameter     | Parameter type | Description                          |
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| id            | url attribute  | Id of the requested cart             |
++---------------+----------------+--------------------------------------+
 
-Parameters
-~~~~~~~~~~
+Example
+^^^^^^^
 
-shipments[X][method]
-    The id of the shipping method, where X is the shipment number.
-    Leave empty to add new
+To check available shipping methods for previously addressed cart, one could use following command:
```

Comment:
`the previously addressed cart`

**the** following command (Before `following` you are **always** missing _the_ 😭 )